doc/METADATA.ini

```
@@ -828,6 +828,12 @@ status = implemented
 usedby/plugin= mathcheck
 description= check if value conforms to mathematical expression
 
+[check/email]
+type= string
+status = implemented
+usedby/plugin= email
+description= check if the given email address is valid based on regular expression
```

how it is implemented is an implementation detail:

```suggestion
description= check if value contains a valid email address
```